### ISSUE TITLE
Calling "flutter doctor --android-licenses" results with "Could not find an option named "android-licenses""

### ISSUE THREAD

**Issue by orestesgaolin** [leoafarias/fvm#34]
**Describe the bug**
Running `flutter doctor --android-licenses` results with error `Could not find an option named "android-licenses"`

**To Reproduce**
1. Install fvm and do not install Flutter from official website
2. Run `fvm use master` for the first time
3. See the flutter doctor's output:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.12.17-pre.67, on Mac OS X 10.15.1 19B2093, locale pl-PL)
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    ✗ Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[✓] Xcode - develop for iOS and macOS (Xcode 11.2.1)
[!] Android Studio (version 3.5)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.40.2)
[!] Connected device
    ! No devices available
```

4. Run `fvm flutter doctor --android-licenses`
5. See error: `Could not find an option named "android-licenses"`
6. Run `./fvm run`
7. Run `flutter doctor --android-licenses`
8. See error `zsh: command not found: flutter`

**Expected behavior**
Flutter doctor should run and allow to accept Android licenses.

**Desktop (please complete the following information):**
 - OS: macOS
 - no standalone Flutter distribution installed 

**Additional context**
It happened on clean install of macOS

**Comment by orestesgaolin** [comment]
The same problem occurs when running `flutter config --enable-macos-desktop`

**Comment by orestesgaolin** [comment]
Ok, the workaround for that is just to `cd` to the fvm fluter directory and execute e.g. `./flutter doctor --android-licenses`.

**Comment by leoafarias** [comment]
@orestesgaolin this is an ongoing issue with flutter command. I have a pull request to the arts package but it has not been approved and a PR I can’t merge. #27 that or why they work around is documented. Thanks for updating the ticket

**Comment by leoafarias** [comment]
Fixed by #62
